launcher/game/web.rpy

```diff
@@ -338,8 +361,9 @@ init python:
 
         # Use re to slugify the game name, avoiding use of 3rd party libraries
         slugified_name = re.sub(r'\W+', '-', p.dump['build']['display_name']).lower()
+        version = "{}-{}".format(slugified_name, int(time.time()))
         # Replace the default cache name with the game name + current timestamp
-        service_worker = service_worker.replace('renpy-web-game', "{}-{}".format(slugified_name, int(time.time())))
+        service_worker = service_worker.replace('renpy-web-game', version)
```

Comment:
I would like to seperate app name and version in service worker, and seperately replace these two values in build.
in service worker:
```
var APP_PREFIX = '||GAME_NAME||'
var VERSION = '||GAME_VERSION||'
var CACHE_NAME = APP_PREFIX + VERSION;
```
And in build:
```
service_worker = service_worker.replace('||GAME_NAME||', slugified_name)
service_worker = service_worker.replace('||GAME_VERSION||', version)
```
And we need a way to get version of service worker in main webpage. I think we can do like this.
```
if (event.request.url.indexOf("getVersionWorker") > 0) {
    event.respondWith(new Response(VERSION));
    return;
}
```
So we can get version just like this.
```
let currentVersion = await (await fetch("getVersionWorker")).text();
```

Reply:
The version is just a timestamp of when the game was built, not the version of the game, so not sure if we need to separate them